MediaPlayer/Times.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import kind from '@enact/core/kind';
+
+import Skinnable from '../Skinnable';
+
+import css from './Times.module.less';
+
+/**
+ * Agate-styled formatted time component.
+ *
+ * @class Times
+ * @memberof agate/MediaPlayer
+ * @ui
+ * @public
+ */
+const TimesBase = kind({
+	name: 'Times',
+
+	styles: {
+		css,
+		className: 'times'
+	},
+
+	render: ({...rest}) => {
+		return (
+			<div {...rest}>
+				<time>2:40</time>
+				<time>-1:28</time>
+			</div>
+		);
+	}
+});
+
+const Times = Skinnable(TimesBase);
```

Comment:
Shouldn't this `Skinnable` move to `MediaPlayer`?

Reply:
`Times.module.less` uses 2 variables for times label positioning. If I remove `Skinnable` from here, the layout would break. Any suggestion on how to fix this is welcomed :)

Reply:
I'm suggesting move `Skinnable` to `MediaPlayer`.
Since `MediaPlayer` uses this module, it will work. You can refer to moonstone's VideoPlayer.

Reply:
Oh, okay. I think I got it now.